src/Interpreters/Streaming/CalculateDataStreamSemantic.cpp

```diff
@@ -13,53 +13,77 @@ DataStreamSemanticEx calculateDataStreamSemanticForJoin(
     DataStreamSemanticEx left_input_data_stream_semantic,
     DataStreamSemanticEx right_input_data_stream_semantic,
     std::pair<JoinKind, JoinStrictness> kind_and_strictness,
+    bool right_input_is_streaming,
     SelectQueryInfo & query_info)
 {
-    HashJoin::validate(
```

Comment:
Why remove the validation here?

Reply:
The func is used for validate ` stream join stream `, so skipped here. and it's already checked in `Streaming::HashJoin::init()`